microcosm_postgres/tests/test_company_store.py

```diff
@@ -98,25 +98,6 @@ def test_create_delete_company(self):
             raises(ModelNotFoundError, pattern="Company not found"),
         )
 
-    def test_create_delete_company_complicated_expression(self):
-        """
-        Delete should support more complicated criterion with the `fetch` synchronization strategy enabled.
-
-        """
-        with transaction():
-            company = Company(name="name").create()
-            Company(name="other-name").create()
-
-        with transaction():
-            self.company_store._delete(Company.name.in_(["name"]), synchronize_session="fetch")
```

Comment:
It turns out the sharded query won't work with "fetch", I can't find anywhere that uses it, so this is removed